projects/plugins/jetpack/tools/check-block-assets.php

```diff
@@ -43,6 +43,7 @@
 	'wp-html-entities',
 	'wp-i18n',
 	'wp-is-shallow-equal',
+	'wp-plugins',
```

Comment:
Did you add this in for the Story block? If we need to add this in here, maybe we could allow it just for the Story block, with a change to `$allowed`?